src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let totalCost = days * costPerDay;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
Instead of using 'else if' after an 'if' with a 'return', you can simplify the code by using only 'if' statements with 'return'. This will make the code cleaner and easier to read.